src/test/scala/cromwell/binding/NestedScatterSpec.scala

```diff
@@ -28,7 +28,7 @@ class ScatterSpec extends FlatSpec with Matchers {
   }
 
   it should "Have 'Scatter' objects indexed properly" in {
-    namespace.workflow.scatters(0).index shouldEqual 0
+    namespace.workflow.scatters.head.index shouldEqual 0
```

Comment:
Yes I actually find it more logical to go with (0) when you're testing several indexes in a row like here.
Maybe it would be worth making a list of stuff in intelliJ highlighting that we don't like (not this in particular but I remember a few other comments about this), configure it once and for all and share the conf file.